include/dxc/DxilContainer/RDAT_Macros.inl

```diff
@@ -72,7 +72,7 @@
   #define RDAT_UNION_END()                    };
   #define RDAT_RECORD_REF(type, name)         RecordRef<type> name;
   #define RDAT_RECORD_ARRAY_REF(type, name)   RecordArrayRef<type> name;
-  #define RDAT_RECORD_VALUE(type, name)       type name;
+  #define RDAT_RECORD_VALUE(type, name)       struct type name;
```

Comment:
Oh, I think there may be another place this should go, but I guess it's not hit by anything right now.  Same change under `DEF_RDAT_TYPES_BASIC_STRUCT` block.